test/javascript.js

```diff
@@ -28,6 +28,21 @@ describe('javascript', function() {
     assert.equal(output.default(), 3);
   });
 
+  it('should produce a basic JS bundle with object rest spread support', async function() {
+    let b = await bundle(__dirname + '/integration/object-rest-spread/object-rest-spread.js');
```

Comment:
It wasn't bundling anything, got it it bundle correctly in the test now, not sure how this worked locally. Now it still throws an error though.

Hope you can resolve it :)